appium/tests/screenobjects/attachment.screen.ts

```diff
@@ -25,7 +25,7 @@ class AttachmentScreen extends BaseScreen {
   }
 
   checkDownloadPopUp = async (name: string) => {
-    ElementHelper.waitElementVisible(await this.cancelButton);
+    await ElementHelper.waitElementVisible(await this.cancelButton);
     const attachment = `-ios class chain:**/XCUIElementTypeNavigationBar[\`name == "com_apple_DocumentManager_Service.DOCServiceTargetSelectionBrowserView"\`]/XCUIElementTypeButton/XCUIElementTypeStaticText`; //it works only with this selector
     expect(await (await $(attachment)).getValue()).toEqual(name);
   }
```

Comment:
What IDE are using for typescript? When you open this project in `vscode`, ensuring you open it as a workspace and not just as a folder, it will highlight these problems automatically.

<img width="1316" alt="image" src="https://user-images.githubusercontent.com/6306961/165487170-ac346afe-2b4b-4ad0-9d3e-9b96c9f16afc.png">

Reply:
I'm using VS Code, but project was opened not as a workspace, so these errors weren't highlighted.
Will open as a workspace next time, thanks

Reply:
@tomholub Do you know solution for opening VSCode workspace file in Jetbrains?

Reply:
Probably it's possible to create WebStorm project in the same folder, so you'll be able to use this functionality too.

Reply:
WebStorm doesn't recognize tsconfig.eslint.json path as it's not relative path.

![image](https://user-images.githubusercontent.com/77344191/165489533-a8bb1234-2a63-4811-81b9-7495bd23f06e.png)
![image](https://user-images.githubusercontent.com/77344191/165489581-92b6cec4-bceb-4fa1-b487-224c99134999.png)

Reply:
I also like Jetbrains but I don't recommend to use WebStorm at all for this project.

We've spent half a decade improving our vscode tooling to catch and display things like this reliably, and it relies on plugins that are specific to vscode and even more specific settings of those plugins. Doing the same in WebStorm, to be consistent with vscode, would be hard.

Reply:
Ok, got it.
I still want to use Webstorm as there are many features Jetbrains only has but let me use VSCode for this project.